opentreemap/treemap/css/sass/modules/_checkboxes.scss

```diff
@@ -1,8 +1,9 @@
 @mixin checkboxes {
-	input[type='radio'],
+    input[type='radio'],
     input[type='checkbox'] {
         opacity: 0;
-        float: left;
+        position: fixed;
+        left: -100px;
         width: 0;
 
```

Comment:
I'm fairly certain that if instead of doing the changes you made here, you add `position: absolute;` the problem will be solved.